Flaky integration test TestDelayingQueueInvariants

https://devops-ci.elastic.co/job/cloud-on-k8s-pr/4225/testReport/junit/github/com_elastic_cloud-on-k8s_pkg_controller_license/Run_tests_in_parallel___Run_unit_and_integration_tests___TestDelayingQueueInvariants_no_dedup_ing_when_delaying/

```
=== RUN   TestDelayingQueueInvariants/no_dedup'ing_when_delaying
    license_controller_integration_test.go:226: 
        	Error Trace:	license_controller_integration_test.go:226
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 1
        	Test:       	TestDelayingQueueInvariants/no_dedup'ing_when_delaying
    --- FAIL: TestDelayingQueueInvariants/no_dedup'ing_when_delaying (0.02s)
```

--- a/pkg/controller/license/license_controller_integration_test.go
+++ b/pkg/controller/license/license_controller_integration_test.go
@@ -174,10 +174,10 @@ func TestDelayingQueueInvariants(t *testing.T) {
 			name: "no dedup'ing when delaying",
 			adds: func(q workqueue.DelayingInterface) {
 				q.Add(item)
-				q.AddAfter(item, 1*time.Millisecond)
+				q.AddAfter(item, 500*time.Millisecond)
 			},
 			expectedObservations: 2,
-			timeout:              10 * time.Millisecond,
+			timeout:              1 * time.Second,
 		},
 		{
 			name: "but dedup's and updates item within the wait queue",

